### PR TITLE
Enable build of crosscomponent for armel

### PR DESCRIPTION
**Pull request by hqueue**
Enable build of crosscomponent for `armel` in addition to `arm`

### CODE REVIEW

**Comment by hqueue** [comment]
@hseok-oh Is this enough to enable building cross component for armel ?

**Comment by hseok-oh** [comment]
We cannot use these components yet. 
related pr: #10159

**Comment by hqueue** [comment]
@hseok-oh Thanks for the feedback.  I understood that it's not working yet. I will check it more.

File: build.sh [hunk 1]
```diff
@@ -283,7 +283,7 @@ build_cross_arch_component()
     __SkipCrossArchBuild=1
     TARGET_ROOTFS=""
     # check supported cross-architecture components host(__HostArch)/target(__BuildArch) pair
-    if [[ "$__BuildArch" == "arm" && "$__CrossArch" == "x86" ]]; then
+    if [[ ("$__BuildArch" == "arm" || "$__BuildARch" == "armel") && "$__CrossArch" == "x86" ]]; then
```
Comment (jkotas) [inline build.sh]: Casing typo: BuildARch

Comment (hqueue) [inline build.sh]: Thanks!

**Comment by BruceForstall** [comment]
@hseok-oh @hqueue Is this ready to merge?

cc @jkotas

**Comment by hseok-oh** [comment]
@BruceForstall I think there is no problem to merge it because the default build and ci build will be not changed.

**Pull request merged**